tests/models/openoil/test_oillibrary.py

```diff
@@ -0,0 +1,348 @@
+            o.set_config('environment:fallback:land_binary_mask', 0)
+            o.set_config('environment:fallback:x_wind', windspeed)
+            o.set_config('environment:fallback:y_wind', 0)
+            o.set_config('environment:fallback:x_sea_water_velocity', 0)
+            o.set_config('environment:fallback:y_sea_water_velocity', .3)
+            o.run(duration=timedelta(hours=3), time_step=900)
+
+            b = o.get_oil_budget()
+            actual_dispersed = b['mass_dispersed'] / b['mass_total']
+            actual_submerged = b['mass_submerged'] / b['mass_total']
+            actual_evaporated = b['mass_evaporated'] / b['mass_total']
+            print('Dispersion fraction %f for '
+                  '%s and wind speed %f' %
+                  (actual_dispersed[-1], oil, windspeed))
+            if oil == 'SMORBUKK KONDENSAT' and windspeed == 3:
+                fraction_dispersed = 0
+                fraction_submerged = 0
+                fraction_evaporated = 0.589
+                meanlon = 4.81742
+            elif oil == 'SMORBUKK KONDENSAT' and windspeed == 8:
+                fraction_dispersed = 0.092
+                fraction_submerged = 0.390
+                fraction_evaporated = 0.473
+                meanlon = 4.809
+            elif oil == 'SKRUGARD' and windspeed == 8:
+                fraction_dispersed = 0.139
+                fraction_submerged = 0.481
+                fraction_evaporated = 0.123
+                meanlon = 4.819
+            else:
+                fraction_dispersed = -1  # not defined
+
+            np.testing.assert_almost_equal(actual_dispersed[-1],
+                                           fraction_dispersed, 2)
+            np.testing.assert_almost_equal(actual_submerged[-1],
+                                           fraction_submerged, 2)
+            np.testing.assert_almost_equal(actual_evaporated[-1],
+                                           fraction_evaporated, 2)
+            np.testing.assert_almost_equal(np.mean(o.elements.lon), meanlon, 3)
+            #o.plot_oil_budget()
+
+
+def test_no_dispersion():
+    o = OpenOil(loglevel=50, weathering_model='noaa')
+
+    o.seed_elements(lon=4.8,
+                    lat=60,
+                    number=100,
+                    time=datetime.now(),
+                    oiltype='SIRTICA')
+    o.set_config('processes:dispersion', False)
+    o.set_config('environment:fallback:land_binary_mask', 0)
+    o.set_config('environment:fallback:x_wind', 8)
+    o.set_config('environment:fallback:y_wind', 8)
+    o.set_config('environment:fallback:x_sea_water_velocity', 0)
+    o.set_config('environment:fallback:y_sea_water_velocity', .3)
+    o.run(duration=timedelta(hours=2), time_step=1800)
+    b = o.get_oil_budget()
+    actual_dispersed = b['mass_dispersed'] / b['mass_total']
+    np.testing.assert_almost_equal(actual_dispersed[-1], 0)
+    np.testing.assert_array_almost_equal(o.elements.lon[0:3], [4.801, 4.801, 4.801], 3)
```

Comment:
@knutfrode: her endte det opp med at alle elementa får samme slutt-longitude. det var ikkje tilfellet før. har du nokon ide om koffor?

Reply:
I get the following figures if I insert the following lines before the assert:
```
o.plot_property('z')
o.plot()
```

![Figure_2](https://user-images.githubusercontent.com/5222051/130794624-7cbadf67-0aa8-4281-ac90-62958454aed7.png)
![Figure_1](https://user-images.githubusercontent.com/5222051/130794629-c9b8c562-122c-4ea7-aa4a-e07bc91d53cf.png)

I suspect that you probably don't see any entrainment, assumedly because viscosity is so high that there is no entrainment(?)